### PR TITLE
Fix illegible rich links on paid articles

### PR DESCRIPTION
**Pull request by JonNorman**
### What does this change?
It's really good when people can read our rich-links on articles. On paid-content articles, this isn't currently possible due to the grey background:

image

These changes make it possible:
image

### Additional Notes
Originally looking into this, I thought I'd need to add a class to the rich-link based on whether the content type had branding. It turns out I didn't need to do this, but did a tiny amount of refactoring of the commercial metadata properties while I was at it. Yes, it probably shouldn't be in this PR, but the changes are minor and it _is_ Friday....

### Request for comment
@guardian/commercial-dev

### CODE REVIEW

**Comment by JonNorman** [comment]
[Illegibile rich links on paid containers](https://trello.com/c/IvVZm6Tq/149-illegibile-rich-links-on-paid-containers)

File: common/app/model/meta.scala [hunk 1]
```diff
@@ -21,45 +22,26 @@ import play.twirl.api.Html
...
-  def make(metadata: MetaData, tags: Tags, apiContent: contentapi.Content): model.Commercial = {
-    make(metadata, tags, Some(apiContent))
-  }
-
-  def make(metadata: MetaData, tags: Tags): model.Commercial = {
-    make(metadata, tags, None)
-  }
-
-  def make(section: Section): model.Commercial = {
+  def make: model.Commercial = {
```
Comment (kelvin-chappell) [inline common/app/model/meta.scala]: This could be a `val` called `empty` or something, but perhaps it's not even needed.

**Review by kelvin-chappell — APPROVED** [review]
Just a small issue. But otherwise 👍

File: common/app/model/meta.scala (hunk 1 above)
Comment (JonNorman) [inline common/app/model/meta.scala]: I've made it empty for now - if it can be removed then that can be done in a separate PR.

**Pull request merged**